kuma/search/templates/search/base.html

```diff
@@ -4,9 +4,6 @@
 {% if not styles %}
   {% set styles = ('search',) %}
 {% endif %}
-{% if not scripts %}
-  {% set scripts = ('search',) %}
-{% endif %}
```

Comment:
Interesting. Could you say more about this? I'm wondering why we checked for the non-existence of `scripts` in the first place.